### DISCUSSION TITLE
How can I handle exceptions?

### DISCUSSION THREAD

**Discussion by Fatpandac** [flet-dev/flet#614] [Q&A]
I want to catch exceptions and then show exceptions on display or let the user send me feedback.

**Comment by FeodorFitsner** [comment]
What kind of exception is that?

**Reply by Fatpandac** [reply]
<img width="820" alt="image" src="https://user-images.githubusercontent.com/30423976/203240969-94f30ce5-bfe9-4689-a6c2-88461efc133b.png">

Any exception when Flet is run.
Can I customize this page?

**Reply by FeodorFitsner** [reply]
Oh, this! I don't think it's customizable at the moment, but I'm going to create a new issue for that.

**Reply by FeodorFitsner** [reply]
https://github.com/flet-dev/flet/issues/619

**Reply by Fatpandac** [reply]
Thanks so much. ❤️
